# DP: Make glibc compile with /bin/sh being dash and not bash.

--- csu/Makefile	2005-06-15 20:59:25.000000000 +0300
+++ csu/Makefile	2005-06-15 20:59:36.000000000 +0300
@@ -241,7 +241,7 @@
 	 esac; \
 	 files="$(all-Banner-files)";				\
 	 if test -n "$$files"; then				\
-	   echo "\"Available extensions:\\n\"";			\
+	   printf '"Available extensions:\\n"';			\
 	   sed -e '/^#/d' -e 's/^[[:space:]]*/	/'		\
 	       -e 's/^\(.*\)$$/\"\1\\n\"/' $$files;		\
 	 fi) > $@T
